### PR TITLE
feat: detect test commands stranded in stopped containers

### PR DESCRIPTION
**Pull request by carlsverre**
validate now inspects every compose container, not just running ones. compose ps returns a PsListing carrying per-container stopped state (via compose ps -a where the backend supports it), and discover_scripts fails with a clear diagnostic when a service defines test commands but its container has exited, since Antithesis can't run commands in stopped containers. Native podman-compose opts out of -a (it rejects the flag or leaks containers from other projects), so the diagnostic is suppressed there.

extract_test_templates replaces container_cp: for running containers it does an unambiguous 'test -d' pre-flight, and for stopped ones it falls back to matching cp stderr to distinguish a missing templates directory from a real copy failure. Per-container scratch dirs keyed on container id support scaled replicas.

Also rename user-facing 'scripts' / 'Test Composer scripts' to 'test commands' throughout, and attach an Antithesis-support suggestion when a launch request gets an empty response body (older server omitting run_id).

### CODE REVIEW

File: src/validate.rs [hunk 1]
```diff
@@ -376,63 +377,172 @@ fn contains_setup_complete(reader: &mut (impl std::io::Read + std::io::Seek)) ->
...
+                result.scripts.len()
+            );
+            if container.stopped && listing.includes_stopped {
+                stopped_with_scripts.insert(service_name.to_string());
+            }
+            all_scripts.extend(result.scripts);
+        }
+    }
+
+    // If every container failed to surrender its templates and nothing
```
[the remark this answers is not in the document]
Comment (carlsverre) [reply src/validate.rs:474]: Fixed in the latest push: `discover_scripts` now dedupes scripts by `(service, test_name, command_name)` as it extends `all_scripts`, so replicas no longer inflate the `Found X first, ...` counts. Per-container stopped-replica detection (`stopped_with_scripts`) is unchanged since it still runs for every container before the dedupe.
